websites/A/AtCoder/dist/metadata.json

```diff
@@ -0,0 +1,30 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.3",
+  "service": "AtCoder",
+  "author": {
+    "name": "Zai",
+    "id": "90431685472038912"
+  },
+  "contributors": [
+    {
+      "name": "AD",
+      "id": "511232034346369055"
+    }
+  ],
```

Comment:
```suggestion
```

Reply:
he helped me create this xd

Reply:
Still please delete, if he would make pull request it would be ok

Reply:
oh oki